app/MyApp/app/views/ListCollectionView.js

```diff
@@ -74,7 +74,7 @@ $(function() {
                 this.model.set({
                     'show': false
                 })
-                this.model.save({
+                this.model.destroy({
```

Comment:
-this.model.save({
whille deleting collection it was only deleting from List collectionvie, not from list collection DB so i changed it with  this.model.destroy({
also to delete from DB, but after counseling with Laxam bro he said I dont have to delete form DB, so this.model.destroy function will be removed